src/Fixer/Phpdoc/PhpdocAlignFixer.php

```diff
@@ -305,40 +354,47 @@ private function fixDocBlock(DocBlock $docBlock): void
                 if ($hasStatic) {
                     $line .=
                         $this->getIndent(
-                            $tagMax - \strlen($item['tag']) + 1,
-                            '' !== $item['static'] ? 1 : 0
+                            $tagMax - \strlen($item['tag']) + $spacingForTag,
+                            '' !== $item['static'] ? $spacingForTag : 0
                         )
                         .('' !== $item['static'] ? $item['static'] : $this->getIndent(6 /* \strlen('static') */, 0));
-                    $hintVerticalAlignIndent = 1;
+                    $hintVerticalAlignIndent = $spacingForTag;
                 } else {
-                    $hintVerticalAlignIndent = $tagMax - \strlen($item['tag']) + 1;
+                    $hintVerticalAlignIndent = $tagMax - \strlen($item['tag']) + $spacingForTag;
                 }
 
                 $line .=
                     $this->getIndent(
                         $hintVerticalAlignIndent,
-                        '' !== $item['hint'] ? 1 : 0
+                        '' !== $item['hint'] ? $spacingForTag : 0
                     )
                     .$item['hint'];
 
                 if ('' !== $item['var']) {
                     $line .=
-                        $this->getIndent((0 !== $hintMax ? $hintMax : -1) - \strlen($item['hint']) + 1)
+                        $this->getIndent((0 !== $hintMax ? $hintMax : -1) - \strlen($item['hint']) + $spacingForTag, $spacingForTag)
                         .$item['var']
                         .(
                             '' !== $item['desc']
-                            ? $this->getIndent($varMax - \strlen($item['var']) + 1).$item['desc']
+                            ? $this->getIndent($varMax - \strlen($item['var']) + $spacingForTag, $spacingForTag).$item['desc']
                             : ''
                         );
                 } elseif ('' !== $item['desc']) {
-                    $line .= $this->getIndent($hintMax - \strlen($item['hint']) + 1).$item['desc'];
+                    $line .= $this->getIndent($hintMax - \strlen($item['hint']) + $spacingForTag, $spacingForTag).$item['desc'];
                 }
 
                 $docBlock->getLine($current + $j)->setContent($line.$lineEnding);
             }
         }
     }
 
+    private function spacingForTag(?string $tag): int
+    {
+        return (\is_int($this->spacing)) ?
+            $this->spacing :
+            ($this->spacing[$tag] ?? $this->defaultSpacing);
```

Comment:
I believe we can go with:

```suggestion
            ($this->spacing[$tag] ?? $this->spacing[self::DEFAULT_SPACING_KEY] ?? self::DEFAULT_SPACING);
```

and get rid of `PhpdocAlignFixer::$defaultSpacing` property and assignment in the constructor.

Reply:
The property has been thrown out!